Add coming soon pages

olyism opened the pull request:
Add "coming soon" pills against Collector pages we'll very soon have content for.

Javascript, JUnit and Swift Collector docs, for instance, are expected to be completed soon. Pytest and Android will take longer and the pills are expected to stay for a while.

The "coming soon" pills can be used on any page. You can add them with the `is_coming_soon` option in`tiles.yml` and/or `sidebar_link_to`.

## Screenshots

### Left sidebar

<img width="358" alt="Screen Shot 2022-06-01 at 1 43 41 am" src="https://user-images.githubusercontent.com/7202667/171214649-94d6beb1-351a-41a3-afe2-bafbea53530f.png">

### Landing page tiles

<img width="810" alt="Screen Shot 2022-06-01 at 1 43 49 am" src="https://user-images.githubusercontent.com/7202667/171214693-6a45c1b1-e3cc-4215-9d28-e61566594112.png">

Note: please remove these pill and link them as content becomes available.

karensawrey reviewed (approved):
Looks good to me! And kudos for updating the Style Guide with the updated info on the beta pills. An approval from me, but I'd also love @plaindocs to take a look at it before this gets merged.

plaindocs commented:
What's up with the preview on this one?

meiqimichelle commented:
This is great! If you have a minute before @plaindocs comes online, do you want to rebase to include the new Features page, and decide how to fit that in your subheading structure?

olyism commented:
Rebased from `main` and rearranged navigation as per recently added content:

2 min Loom 🎦 https://www.loom.com/share/8c0afe938c3d420fb9c101c5eea862ba

Summary of changes:

- Left sidebar
  1. Break up into 3 sections: getting started, how-to guides and collectors
  2. Getting started section holds overview, test suits, monitor and access control pages. This section emulates the 'user manual' experience when you open the packaging of a new product to guide customers on the key features
- Tiles: homepage and TA landing page tile links updated to link to respective pages/anchors.

<img width="268" alt="Screen Shot 2022-06-01 at 12 27 32 pm" src="https://user-images.githubusercontent.com/7202667/171315678-a127f885-0403-4b41-881f-5e1186e0e348.png">

olyism commented:
Pulling back into draft after a huddle with @meiqimichelle and @toolmantim to make further improvements:

1. **Downplay "coming soon"** as it's not actionable
2. Replace with a catch-all 'Other frameworks' page with instructions for unsupported frameworks that directs them to use JSON, XML or write your own collectors

Might keep the 'coming soon' work that's in this PR for now, as it's useful for building out a pill design pattern (e.g. beta, coming soon, new etc).

Another commit to come soon.

olyism commented:
Just pushed up the changes. Ready for review again 🙏

Loom 🎦 https://www.loom.com/share/d8d61b044b8d47d4a28e8f2f6b4dd5b5

Summary of changes:

- Removed all coming soon links
- Added new 'Other languages' page
- Add 'other collectors' link to sidebar and landing page tile

![Screen Shot 2022-06-01 at 4 32 10 pm](https://user-images.githubusercontent.com/7202667/171341996-1ccc42e7-4ae0-4adc-91a1-75031c483018.png)

![Screen Shot 2022-06-01 at 4 32 39 pm](https://user-images.githubusercontent.com/7202667/171342060-f359a628-c989-4238-97b4-fd360b77303c.png)

plaindocs commented:
@olyism that lastest Loom is broken for me. But the content looks good (didn't look into the CSS in detail.

meiqimichelle reviewed (approved):
This looks great!!

One thing to note -- I haven't actually made the test analytics community Slack channel yet (I'm scheduled to do that with JJ tomorrow). So, may want to hold off on shipping this until that channel is live.

olyism commented:
Merged ✅ 🚢

We're working really close to the launch day so I've simply removed that Slack channel sentence and merged it. @meiqimichelle let me know when that channel is up and I'll open a separate PR.

meiqimichelle commented:
Perfection! Can do. Thank you!

lachlanhardy commented on app/assets/stylesheets/public/utils/_pill.scss:
```diff
@@ -0,0 +1,64 @@
+$_pills: (
```
Neat!
